libs/sdk-ui-dashboard/src/model/events/filters.ts

```diff
@@ -11,7 +11,7 @@ import { eventGuard } from "./util";
 /**
  * This event is emitted after the dashboard's date filter selection is changed.
  *
- * @alpha
+ * @public
```

Comment:
Please also document the individual items in payload.

Reply:
Added.